docs/src/main/sphinx/connector/cassandra.md

```diff
@@ -49,6 +49,7 @@ The following configuration properties are available:
 | `cassandra.native-protocol-port` | The Cassandra server port running the native client protocol, defaults to `9042`.                                                                                                                                                                                                                                                                                                                                           |
 | `cassandra.consistency-level`    | Consistency levels in Cassandra refer to the level of consistency to be used for both read and write operations.  More information about consistency levels can be found in the [Cassandra consistency] documentation. This property defaults to a consistency level of `ONE`. Possible values include `ALL`, `EACH_QUORUM`, `QUORUM`, `LOCAL_QUORUM`, `ONE`, `TWO`, `THREE`, `LOCAL_ONE`, `ANY`, `SERIAL`, `LOCAL_SERIAL`. |
 | `cassandra.allow-drop-table`     | Enables {doc}`/sql/drop-table` operations. Defaults to `false`.                                                                                                                                                                                                                                                                                                                                                             |
+| `cassandra.security`             | Should be set to `PASSWORD` for password based authentication. Defaults to `NONE`.                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
Maybe use

```
Configure authentication to Cassandra. Defaults to `NONE`. Set to `PASSWORD` for basic authentication, and configure `cassandra.username` and `cassandra.password`.
```

Also maybe we should redo the two following ones to mention that `cassandra.security=PASSWORD` is required?  

See my following suggestion.